sklearn/linear_model/logistic.py

```diff
@@ -1178,6 +1178,13 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
     Attributes
     ----------
 
+    classes_ : array or shape (n_classes, )
```

Comment:
what is ``array or shape``?

Reply:
from looking at other examples, I am guessing it should be: 
`array, shape (n_classes, )`
Is that correct?

Reply:
IMO `array, shape ...` or `list, shape ...`

Reply:
that's acceptable